spec/models/carto/user_creation_spec.rb

```
@@ -53,6 +53,11 @@
   describe 'validation token' do
     include_context 'organization with users helper'
 
+    before(:each) do
+      # Disable central for these tests
+      Cartodb::Central.stubs(:sync_data_with_cartodb_central?).returns(false)
```

During using creation, the hook is called to sync data with central. Sequel 3 was not properly handling this error and it silently ignored it. The upgrade to Sequel 4 makes this error raise an exception and the tests to fail. They should have been failing the whole time. We just disable central (like other tests in this suite).